tests/notification.c

```diff
@@ -243,6 +243,31 @@ test_bytes_icon ()
   test_icon (serialized_icon_s, NULL, FALSE);
 }
 
+static void
+test_file_icon ()
+{
+  g_autofree char *uri = NULL;
+  g_autofree char *serialized_icon_s = NULL;
+  g_autofree char *exp_serialized_icon_s = NULL;
+  g_autoptr(GVariant) serialized_icon = NULL;
+  g_autoptr(GFile) file = NULL;
+  g_autoptr(GIcon) icon = NULL;
+  g_autoptr(GFileIOStream) iostream = NULL;
+  GOutputStream *stream = NULL;
+
+  file = g_file_new_tmp ("iconXXXXXX", &iostream, NULL);
+  stream = g_io_stream_get_output_stream (G_IO_STREAM (iostream));
```

Comment:
I see. So, the include I mentioned earlier should have been added in this commit, not that other one.